pkg/ottl/ottlfuncs/func_contains_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package ottlfuncs
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl"
+)
+
+func Test_contains(t *testing.T) {
```

Comment:
Can you add a test for the item being a `pcommon.Value`, `pcommon.Map`, and `pcommon.Slice`.  I suspect those cases are not going to work bc we take the item as is.